packages/spruce-next-helpers/src/skillskit/helpers/lang.ts

```diff
@@ -0,0 +1,115 @@
+import * as moment from 'moment'
+import Dinero from 'dinero.js'
+
+import { ICoreLocation } from '@sprucelabs/spruce-types'
+
+export default {
+	lang: {},
+	overrides: {},
+	async configure(langDir: string) {
+		this.lang = require(`${langDir}/default.js`)
+		try {
+			this.overrides = require(`${langDir}/overrides.js`)
+		} catch (err) {
+			console.info('No lang override specified.')
+		}
+	},
+	mixin(lang, overrides) {
+		this.lang = lang
+		this.overrides = overrides
+	},
+	getText(key: string, context: Record<string, any> = {}) {
+		const translations: Record<string, any> = {
+			...this.lang,
+			...this.overrides,
+			...context
+		}
+		if (translations[key]) {
+			return typeof translations[key] === 'function'
+				? translations[key](translations)
+				: translations[key]
+		}
+
+		throw Error(`Translation missing key ${key}`)
+	},
+	/** Converts a numnber of cents into a formatted currency string. Ex: "$23.99" */
+	friendlyCurrency(cents: number, location: ICoreLocation): string {
+		/* TODO: Use these props once added to Location
+		const currency: string = location.currecy || 'USD'
+		const locale: string = location.locale || 'en-US'
+		const precision: number = location.currencyMinorUnits || 2
+		*/
+		console.log('Location passed to friendlyCurrency: ', location.id)
```

Comment:
Remove log; add an eslint ignore next-line for `location` if it complains about that, should be a fix suggestion in vscode.